patches/chrome-android-java-res-layout-bottom_toolbar_browsing.xml.patch

```diff
@@ -1,20 +1,50 @@
 diff --git a/chrome/android/java/res/layout/bottom_toolbar_browsing.xml b/chrome/android/java/res/layout/bottom_toolbar_browsing.xml
-index 76d517641db816c12bd35ae6497cd172eea71fd0..e56cffeee66034c659ac7719899f2512a1851c72 100644
+index 76d517641db816c12bd35ae6497cd172eea71fd0..2d13046d3bdac0e21bb9032d9358728b917ec95a 100644
 --- a/chrome/android/java/res/layout/bottom_toolbar_browsing.xml
 +++ b/chrome/android/java/res/layout/bottom_toolbar_browsing.xml
-@@ -62,6 +62,7 @@
-         android:contentDescription="@string/accessibility_toolbar_btn_tabswitcher_toggle_default"
-         app:menuVerticalOverlapAnchor="false" />
+@@ -16,7 +16,7 @@
+     android:paddingStart="@dimen/bottom_toolbar_padding"
+     android:paddingEnd="@dimen/bottom_toolbar_padding" >
  
+-    <include layout="@layout/toolbar_space" />
++    <!-- <include layout="@layout/toolbar_space" /> -->
```

Comment:
these can be effectively "removed" by creating an empty version of the file in brave/android. I believe @emerick had a PR that did this

Reply:
But we will have to patch this place with new version, right?

Reply:
no, just use the same path and it will just work

Reply:
oops, this was toolbar_space so you're right

Reply:
Should it be changed on
+<!--
     <include layout="@layout/toolbar_space" />
+-->
Or it doesn't matter?